.ci/install_firecracker.sh

```diff
@@ -24,73 +24,31 @@ if [ "$docker_version" != "18.06" ]; then
 	die "Firecracker hypervisor only works with docker 18.06"
 fi
 
-
-
 install_fc() {
 	# Get url for firecracker from runtime/versions.yaml
 	firecracker_repo=$(get_version "assets.hypervisor.firecracker.url")
 	[ -n "$firecracker_repo" ] || die "failed to get firecracker repo"
-	firecracker_repo=${firecracker_repo/https:\/\//}
 
 	# Get version for firecracker from runtime/versions.yaml
 	firecracker_version=$(get_version "assets.hypervisor.firecracker.version")
 	[ -n "$firecracker_version" ] || die "failed to get firecracker version"
 
-	# Get firecracker repo
-	go get -d "${firecracker_repo}" || true
-	# Checkout to specific version
-	pushd "${GOPATH}/src/${firecracker_repo}"
-	git checkout "tags/${firecracker_version}"
-	./tools/devtool --unattended build --release
-	sudo install ${GOPATH}/src/${firecracker_repo}/build/cargo_target/${arch}-*-linux-musl/release/firecracker /usr/bin/
-	sudo install ${GOPATH}/src/${firecracker_repo}/build/cargo_target/${arch}-*-linux-musl/release/jailer /usr/bin/
-	popd
+	# Download firecracker and jailer
+	curl -fsL ${firecracker_repo}/releases/download/v${firecracker_version}/firecracker-v${firecracker_version}-${arch} -o /usr/bin/firecracker
```

Comment:
Please split these commands into two parts:

1) Download as the current user.
2) Run `sudo install` to install them.